opus/application/static_media/js/browse.js

```diff
@@ -572,15 +573,25 @@ var o_browse = {
         let contentsView = o_browse.getScrollContainerClass();
         let maxObs = (opus.prefs.view === "cart" ? o_cart.cartCount : opus.resultCount);
 
-        let obsNum = $(`${tab} .thumbnail-container[data-id=${opusId}]`).data("obs") + 1;
+        let obsNum = $(`${tab} .thumbnail-container[data-id=${opusId}]`).data("obs");
         if (obsNum <= maxObs) {
-            let nextElem = $(`${tab} .thumbnail-container[data-obs=${obsNum}]`);
+            let nextElem = (contentsView === ".op-gallery-view" ? $(`${tab} .thumbnail-container[data-obs=${obsNum}]`) : $(`${tab} tr[data-obs=${obsNum}]`));
```

Comment:
I thought you told Dave you didn't want to compare against the scroll class name? Shouldn't you use something like isGalleryView?

Reply:
yes - missed this, good catch